src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -185,16 +199,32 @@ export class MeetingManager implements AudioVideoObserver {
     configuration: MeetingSessionConfiguration,
     deviceLabels: DeviceLabels | DeviceLabelTrigger = DeviceLabels.AudioAndVideo,
   ): Promise<any> {
-    const { logger: inputLogger, simulcastEnabled, videoDownlinkBandwidthPolicy } = this.config;
-    const logger = inputLogger ? inputLogger : this.createLogger(configuration);
-
+    const {
+      simulcastEnabled,
+      logger: configLogger,
+      logLevel,
+      postLogConfig,
+      videoDownlinkBandwidthPolicy
+    } = this.meetingManagerConfig;
+    
     if (simulcastEnabled) {
+      this.simulcastEnabled = simulcastEnabled;
       configuration.enableUnifiedPlanForChromiumBasedBrowsers = true;
       configuration.enableSimulcastForUnifiedPlanChromiumBasedBrowsers = true;
-    };
+    }
+    if (configLogger) {
+      this.logger = configLogger;
+    } else {
+      this.logLevel = logLevel;
+      if (postLogConfig) {
+        this.postLoggerConfig = postLogConfig;
+      }
+    }
+    const logger = this.logger ? this.logger : this.createLogger(configuration);
```

Comment:
use configLogger instead of this.logger.

Reply:
Updated, thanks!